gradle/formatter.properties

```diff
@@ -1,38 +1,29 @@
 #Whether to use 'space', 'tab' or 'mixed' (both) characters for indentation. 
 #The default value is 'tab'.
 org.eclipse.jdt.core.formatter.tabulation.char=space
-
```

Comment:
Please restore the spaces.  This is an unneeded change.

Reply:
indeed.